editors/boost-up-my-code-editor.md

```diff
@@ -0,0 +1,96 @@
+# [MO] Boost up my code editor
+
+## Owner: [Guillaume Diallo-Mulliez](https://github.com/guitoof)
+
+## ⛳️ Control Points
+
+My editor allows me to:
+
+### 🐞 **Properly Debug**: so that I can quickly identify the responsibility of an unexpected behaviour
+
+##### What ?
+
+- Setup break points
+- Inspect current context variables
+
+##### How ?
+
+- for **VSCode**: [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome)
+
+### 🎨 **Automatically Format my Code**: so that I can follow my team syntactic standards and ease up code review and maintanability
+
+##### What ?
+
+- Real time or On save formatting
+- Inspect current context variables
+
+#### How ?
+
+- for **VSCode**: [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
+
+### 🔍 **Quickly Spot Syntactic Errors**: so that I don't need to run my linter/type checker commands after each modification
+
+##### What ?
+
+- Linter
+- Type checking (flow, typescript)
+
+#### How ?
+
+- for **VSCode**:
+  - for **ESLint** : [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
+  - for **TSLint** : [TSLint](https://marketplace.visualstudio.com/items?itemName=eg2.tslint)
+
+### 📦 **Identify Outdated packages**: so that I can upgrade them rapidly after a new release
+
+##### What ?
+
+- See the latest stable version of each package
+
+#### How ?
+
+- for **VSCode**: [Version Lens](https://marketplace.visualstudio.com/items?itemName=pflannery.vscode-versionlens)
+
+### ☁️ **Visualize my git diff**: so that I can see what I changed since the last time it worked
+
+##### What ?
+
+- See the git diff
+- Revert changes on a file
+
+#### How ?
+
+- for **VSCode**: Build in _Source Control_ tab
+
+### 🛠 **Create & use snippets**: so that I can create components, tests and other basic files in a flash
+
+#### How ?
+
+- for **VSCode**: ???
+
+### 🔮 **Use Autocompletion**: so that I can avoid typos and code faster
+
+##### What ?
+
+- Variables & function names
+- Relative and absolute paths
+
+#### How ?
+
+- for **VSCode**: ???
+
+### ⛵️ **Navigate throughout the code in a click**: so that I don't waste time fumbling through a complex tree view
+
+##### What ?
+
+- Varibales, Functions, Classes **definition**
+- Varibales, Functions, Classes **documentation**
+- Varibales, Functions, Classes **typing**
+
+#### How ?
+
+- for **VSCode**: build in ???
```

Comment:
I guess the `Flow` extension works if you use flow, or vanilla VSCode if you do TypeScript.

I think you're screwed if you do plain javascript (but I should check)